ciao-image/main.go

```diff
@@ -43,8 +46,16 @@ func init() {
 }
 
 func main() {
-	// TBD Select the right datastore interface
-	metaDs := &datastore.Noop{}
+	metaDs := &datastore.MetaDs{
+		DbProvider: database.NewBoltDBProvider(),
+		DbDir:      dbDir,
+		DbFile:     dbFile,
+	}
+	metaDsTables := []string{"images"}
+	metaDs.DbInit(metaDs.DbDir, metaDs.DbFile)
```

Comment:
And ignoring the errors from DbInit and DBTableInit.

A good utility to help you catch all these missing error checks is

https://github.com/kisielk/errcheck

If you know its safe to ignore the error you can explicitly cast it away using _ =